src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -3337,12 +3337,14 @@
 short-summary: Create an address pool.
 parameters:
   - name: --backend-address
-    short-summary: Backend addresses information for backend address pool. If it's used, --vnet is also required.
+    short-summary: Backend addresses information for backend address pool. If it's used, --vnet is required or subnet is required.
     long-summary: |
-        Usage: --backend-address name=addr1 ip-address=10.0.0.1 --vnet MyVnet
+        Usage1: --backend-address name=addr1 ip-address=10.0.0.1 --vnet MyVnet
+        Usage2: --backend-address name=addr1 ip-address=10.0.0.1 subnet==/subscriptions/000/resourceGroups/MyRg/providers/Microsoft.Network/virtualNetworks/vnet/subnets/subnet1
```

Comment:
'Subnet' is the replace of 'vnet', so they should be in the same level. I think you should use '--subnet'.

Reply:
BTW, `subnet` is belong to a `vnet`. So if a user provide a vnet name and a subnet name then you can generate the subnet ID. So `--vnet` and `--subnet` should provide both in the case.